source/templates.tex

```diff
@@ -5331,11 +5335,9 @@
 denoted by a \grammarterm{simple-template-id}
 in which either the template name is a template parameter or any of the
 template arguments is a dependent type or an expression that is type-dependent
-or value-dependent or is a pack expansion
-\begin{note}
-This includes an injected-class-name\iref{class.pre} of a class template
-used without a \grammarterm{template-argument-list}.
-\end{note}%
+or value-dependent or is a pack expansion%
+\footnote{This includes an injected-class-name\iref{class.pre} of a class template
+used without a \grammarterm{template-argument-list}.}
 , or
```

Comment:
Please move the footnote after the punctuation.

Reply:
interesting, didn't know that footnotes should follow punctuation like this, will try to remember for future.

done.